project_4.html

```diff
@@ -0,0 +1,545 @@
+				<li>The circuit must be built on a breadboard.</li>
+			</ul>
+			<h3 class="mb-3" style="display: inline;" id="pj-heading-rgb-dimmer">RGB LED Dimmer</h3>
+			<ul class="mb-5">
+				<li>You must build a circuit with a dimmable RGB LED.</li>
+				<li>The user must be able to adjust the RGB LED to any color when turning the three potentiometer dials.</li>
+				<li>Each potentiometer should control the light intensity of one of the three LED colors (red, green, and blue).</li>
+				<li>When turning a potentiometer clockwise, its respective LED color should brighten.</li>
+				<li>When turning a potentiometer counter-clockwise,  its respective LED color should dim.</li>
+				<li>The circuit must be built on a breadboard.</li>
+			</ul>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
+
+			<!-- ======= Part Table ======= -->
+			<table class="table pj-part-table mb-5">
+				<thead>
+					<tr>
+						<th scope="col">Part Name</th>
+						<th scope="col">Qty</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<td>Jumper Wire</td>
+						<td>X?</td>
+					</tr>
+					<tr>
+						<td>Breadboard</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/project_4/ops_potentiometer.jpg' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								Potentiometer, 10kΩ
+							</a>
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>LED, 3.2V</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/project_4/ops_rgb_led.jpg' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								LED, RGB, Common Cathode
+							</a>
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/parts/ops_part_arduino_nano.png' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								Arduino Nano
+							</a>
+						</td>
+						<td>
+							1
+						</td>
+					</tr>
+					<tr>
+						<td>Resistor, 130Ω</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>Resistor, 430Ω</td>
+						<td>2</td>
+					</tr>
+					<tr>
+						<td>Resistor, 470Ω</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/project_4/ops_micro_usb.jpg' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								Nano-compatible USB Cable
+							</a>
+						</td>
+						<td>1</td>
+					</tr>
+				</tbody>
+			</table>
+			<!-- End Part Table -->
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-schematics">Schematics</h2>
+			<div class="accordion mb-5" id="schemAccordian">
```

Comment:
please check the opening and closing <\div\> tags here. They're not balanced.

Reply:
fixed, I think